source/_posts/2020-07-11-android-18-110-release.markdown

```diff
@@ -0,0 +1,106 @@
+---
+layout: post
+title: "Home Assistant Companion for Android 1.8 - 1.10"
+description: "Today we're releasing a big update for our Android companion app including lots of improvements to notifications, biometrics support and a new geocoded sensor."
+date: 2020-06-18 00:00:00
+date_formatted: "June 18, 2020"
+author: Daniel Shokouhi
+comments: true
+categories: Release-Notes
+og_image: /images/blog/2020-07-android-18-110-release/biometric_unlock.png
+---
+
+There have been lots of more improvements to the android app since we last had a release post so lets go over some of the highlights!
+
+## 1.8.0
+
+Starting with 1.8.0 we have introduced a new [Geocoded sensor](https://companion.home-assistant.io/docs/core/sensors#geocoded-location-sensor) that helps translate your GPS location to an actual street address. We also enhanced our image notifications to allow for not only a relative path (i.e. storing in your [`www`](https://www.home-assistant.io/integrations/http#hosting-files) folder) but you can also request a snapshot using the [Camera Proxy API](https://developers.home-assistant.io/docs/api/rest/#get-apicamera_proxycameraentity_id). All authentication is handled via the app so you don't need to worry about it. You can also use the [click action](https://companion.home-assistant.io/docs/notifications/notifications-basic/#notification-click-action) and [actionable notifications](https://companion.home-assistant.io/docs/notifications/actionable-notifications#building-automations-for-notification-actions) to navigate to a specific lovelace view. [Widgets](https://companion.home-assistant.io/docs/core/android-widgets) were also enhanced to allow you to select from a list of valid services and the ability to add data fields.
```

Comment:
Conder moving all URLs out of the text and use link references in Markdown instead.
The Markdown became really hard to read now.

Reply:
Ok I will update this, maybe what I can do to reduce the amount of references is to link to a single page if the same feature is described on that page.  It did get a little out of hand when I started adding them.